openbb_platform/core/pyproject.toml

```diff
@@ -9,17 +9,17 @@ packages = [{ include = "openbb_core" }]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-uvicorn = "^0.24"
-websockets = "^12.0"
+uvicorn = "^0.32.0"
+websockets = "^13.0"
 pandas = ">=1.5.3"
 html5lib = "^1.1"
-fastapi = "^0.111.0"
+fastapi = "^0.115.0"
 uuid7 = "^0.1.0"
 posthog = "^3.3.1"
 python-multipart = "^0.0.7"
 pydantic = "^2.5.1"
 requests = "^2.32.1"
-importlib-metadata = "^6.8.0"
+importlib-metadata = ">=6.8.0"
```

Comment:
Another question - do we need this pin at all?

Reply:
Not entirely sure, but that should be the minimum version at least because that is when support for 3.12 was added.